ui/ducks/confirm-alerts/confirm-alerts.ts

```diff
@@ -0,0 +1,116 @@
+import { Severity } from '../../helpers/constants/design-system';
+
+export type Alert = {
+  key: string;
+  field?: string;
+  severity: Severity.Danger | Severity.Warning | Severity.Info;
+  message: string;
```

Comment:
We'll want to be explicit about the purpose of the `message` versus the `alertDetails`.